lib/kube/proxy/forwarder.go

```diff
@@ -493,6 +507,45 @@ func (f *Forwarder) setupContext(ctx auth.Context, req *http.Request, isRemoteUs
 	return authCtx, nil
 }
 
+func (f *Forwarder) authorize(ctx context.Context, actx *authContext) error {
+	if actx.teleportCluster.isRemote {
+		// Authorization for a remote kube cluster will happen on the remote
+		// end (by their proxy), after that cluster has remapped used roles.
+		f.WithField("auth_context", actx.String()).Debug("Skipping authorization for a remote kubernetes cluster name")
```

Comment:
nit: does it need to mention `name`? `... for a remote kubernetes cluster.`
Ditto below